packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -339,11 +339,41 @@ const takeScreenshot = (Cypress, state, screenshotConfig, options: TakeScreensho
       return null
     })
     .then(() => {
+      // could fail if iframe is cross-origin, so fail gracefully
+      try {
+        if (disableTimersAndAnimations) {
+          $dom.addCssAnimationDisabler($el)
+        }
+
+        _.each(getBlackout(screenshotConfig), (selector) => {
+          $dom.addBlackouts($el, selector)
+        })
```

Comment:
What is this logic needed for?

Reply:
This used to be done in the runner, but that doesn't work when the AUT is cross-origin, so we do it in the driver now.

Reply:
This logic was moved from here: 
https://github.com/cypress-io/cypress/blob/e216ae2422706d4f46e6dfa68abe2d589c72cf96/packages/runner-shared/src/iframe/aut-iframe.js#L397-L413